src/quenchxx/VaderCookbook.h

```diff
@@ -19,7 +19,7 @@ namespace quenchxx {
 static std::map<std::string, std::vector<std::string>> vaderQuenchxxCustomCookbook() {
   return
     // pt: from t and pkz
-    {{"potential_temperature",        {"AirPotentialTemperature_B"}},
+    {{"airpotential_temperature",        {"AirPotentialTemperature_B"}},
```

Comment:
Is "airpotential_temperature" correct, or should it be "air_potential_temperature" ?